forms/src/App.css

```
@@ -0,0 +1,9 @@
+.survey-creation-name-container {
```

survey creation name contianer? that's from survey creation code.
please, rename the class.
```suggestion
.survey-submit-container {
```